packages/db/prisma/migrations/20260630090000_add_template_community_tables/migration.sql

```
@@ -0,0 +1,57 @@
+
+-- AddForeignKey
+ALTER TABLE "template_reviews" DROP CONSTRAINT IF EXISTS "template_reviews_templateId_fkey";
+ALTER TABLE "template_reviews" ADD CONSTRAINT "template_reviews_templateId_fkey" FOREIGN KEY ("templateId") REFERENCES "templates"("id") ON DELETE CASCADE ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "template_reviews" DROP CONSTRAINT IF EXISTS "template_reviews_userId_fkey";
+ALTER TABLE "template_reviews" ADD CONSTRAINT "template_reviews_userId_fkey" FOREIGN KEY ("userId") REFERENCES "users"("id") ON DELETE CASCADE ON UPDATE CASCADE;
+
+-- AddForeignKey
+ALTER TABLE "user_owned_templates" DROP CONSTRAINT IF EXISTS "user_owned_templates_userId_fkey";
+ALTER TABLE "user_owned_templates" ADD CONSTRAINT "user_owned_templates_userId_fkey" FOREIGN KEY ("userId") REFERENCES "users"("id") ON DELETE CASCADE ON UPDATE CASCADE;
```

SQLFluff (4.2.2)

[error] ADD CONSTRAINT ... FOREIGN KEY should use NOT VALID to avoid locking the table while validating existing rows.

(PG01)